src/components/GagesBarChartAnimation.vue

```diff
@@ -3866,14 +3866,14 @@
             class="text-big"
             transform="translate(100 1017.2)"
           >
-            That data is critical <tspan
+            These data are critical <tspan
               x="0"
               y="28.8"
-            >for</tspan> <tspan font-weight="700">water management.</tspan>
+            >for</tspan> <tspan font-weight="700">water decisions.</tspan>
```

Comment:
Perhaps, "water management decisions". "water decisions" doesn't sound like a thing but maybe it is because I hear the phrase "water management" all the time.